Core/include/Acts/Seeding/EstimateTrackParamsFromSeed.hpp

```diff
@@ -214,24 +214,33 @@ std::optional<BoundVector> estimateTrackParamsFromSeed(
   Vector3 local1 = transform.inverse() * spGlobalPositions[1];
   Vector3 local2 = transform.inverse() * spGlobalPositions[2];
 
-  // Lambda to transform the coordinates to the (u, v) space
-  auto uvTransform = [](const Vector3& local) -> Vector2 {
-    Vector2 uv;
-    ActsScalar denominator = local.x() * local.x() + local.y() * local.y();
-    uv.x() = local.x() / denominator;
-    uv.y() = local.y() / denominator;
-    return uv;
-  };
-  // The uv1.y() should be zero
-  Vector2 uv1 = uvTransform(local1);
-  Vector2 uv2 = uvTransform(local2);
+  // In the new frame the bottom sp is at the origin, while the middle
+  // sp in along the x axis. As such, the x-coordinate of the circle is
+  // at: x-middle / 2.
+  // The y coordinate can be found by using the straight line passing
+  // between the mid point between the middle and top sp and tangent to
+  // the line connecting them
+  Vector2 circleCenter;
+  circleCenter(0, 0) = 0.5 * local1(0, 0);
```

Comment:
Any reason for the 2D access syntax for a 1D vector?

Reply:
my ignorance on how Eigen works ... no idea I could use only one number ...